### ISSUE TITLE
Hard to tell what is locked

### ISSUE THREAD

**Issue by spikefishjohn** [GNS3/gns3-gui#3279]
**Is your feature request related to a problem? Please describe.**
There is no indication on a project what is locked. Maybe a lock icon could be added next to the green icon in the tray on the right?

**Describe the solution you'd like**
Just some way to tell what is locked.

**Describe alternatives you've considered**
Trying to delete a locked node will throw a useful message. 

**Additional context**

**Comment by grossmj** [comment]
What about something similar to View -> Show layers?

![Screenshot from 2022-01-16 21-05-09](https://user-images.githubusercontent.com/4752614/149656539-5400c15d-74a4-4979-8a10-c7e96cdd8630.png)

or I could add an extra column in the topology summary?

**Comment by spikefishjohn** [comment]
Yeah if those red "1"s were maybe a lock icon that would be great! I still think something on topology view would be useful as well, but this works.

**Comment by cristian-ciobanu** [comment]
I'm also creating large topologies with 100+ devices and sometimes I manually unlock a node to edit it.

I agree this feature can help a lot to identify which nodes are locked and which aren't.

Currently I see that when I use the option View --> Lock or unlock all items, the menu icon changes states from a lock icon to an unlock icon and back,

To have a menu option something like View --> Nodes status (Locked/Unlocked) and display a small red lock/ green unlock icon on the node would work better similar to View --> Show layers.
